### ISSUE TITLE
How do I debug provisioning on servers created from the rancher ui

### ISSUE THREAD

**Issue by rlewkowicz-rcd** [rancher/rancher#33375]
I have another issue open, and I dont mind if I do or do not get help with it. I would love someone to teach me how to fish though. 

I get stuck waiting on ssh. How do I see whats actually happening? I can go look at the pod logs etc, I just need to know where I'm looking besides these non de-script status messages. 

I want to make sure rancher is actually mounting this iso I've read is being created for vsphere

**Comment by rlewkowicz-rcd** [comment]
So I resolved my issue, but it'd still be nice to have a way to debug this stuff
